frontend/src/components/MyLoginForm.tsx

```diff
@@ -66,27 +66,23 @@ const MyLoginForm = () => {
   const classes = stylesInUse()
   const [showFormStatus, setShowFormStatus] = useState(false)
 
-  const [loginUser, { loading: loginLoading, data: loginData }] = useMutation(
-    LOGIN
-  )
+  const [loginUser] = useMutation(LOGIN)
 
   const logInUser = async (data: LoginFormFields, resetForm: Function) => {
     try {
-      await loginUser({
+      const loginResponse = await loginUser({
         variables: {
           username: data.username,
           password: data.password,
         },
       })
       setFormStatus(formStatusProps.success)
+      localStorage.setItem('token', loginResponse.data.login.token)
+      window.location.href = '/'
```

Comment:
should we use react router redirect hire?

Reply:
Sounds like a more react way of doing things. Tried to use the `useHistory` hook to redirect but only ran into some errors with react state. Do you a solution in mind?

Reply:
https://dev.to/projectescape/programmatic-navigation-in-react-3p1l  <- that blogs 1st method could work. But making new state and condition just for redirect might be too much hassle

Reply:
Tried to do this. Didn't get it to work and it needed a lot of changes. Biggest problem was that query(ME) didn't refetch after redirect. Lets keep the old  `window.location.href = '/'`

Reply:
Does tulma95's comments mean that current version is approved? I am asking this because pull request is still lacking of one approval.